compiler/resolution/functionResolution.cpp

```diff
@@ -5798,6 +5807,101 @@ static void resolveNewHandleNonGenericInitializer(CallExpr*      call,
   }
 }
 
+// Similar to resolveNewHandleGenericInitializer, except we have been
```

Comment:
I am pretty sure there's something wrong with the tail of this comment i.e.
"and then double check that the type we were given".  Not sure if there's
text missing or if the word "that" is extraneous.

Reply:
I think I forgot to finish that sentence, thanks!